wordpress-mobile/WordPress-iOS
# Allow to set site privacy on the app.

Allows the user to change their site privacy trough the application.
This is only available for WP.com site because self-hosted site using the default WP code don't have this option available on XML-RPC.

How to test:

* Open site ->settings
* Change the values and see if it reflects on the web.

Needs Review: @jleandroperez
